cpp/include/ray/api/task_options.h

```diff
@@ -45,11 +45,19 @@ enum class PlacementStrategy {
   UNRECOGNIZED = -1
 };
 
+enum PlacementGroupState {
+  PENDING = 0,
+  CREATED = 1,
+  REMOVED = 2,
+  RESCHEDULING = 3,
+};
+
 struct PlacementGroupCreationOptions {
   bool global;
   std::string name;
   std::vector<std::unordered_map<std::string, double>> bundles;
   PlacementStrategy strategy;
+  PlacementGroupState state;
```

Comment:
Why add this state to `PlacementGroupCreationOptions `? It's creation options, and I think we can't add this state.

Reply:
Right, the state should be internal